Document/0x05g-Testing-Network-Communication.md

```diff
@@ -284,18 +288,32 @@ Create an SSLContext that uses the TrustManager
 sslContext.init(null, tmf.getTrustManagers(), null);
 ```
 
-#### Black-box Testing
+#### Dynamic Analysis
 
-Black-box Testing can be performed by launching a MITM attack using your prefered Web Proxy to intercept the traffic exchanged between client (mobile application) and the backend server. If the Proxy is unable to intercept the HTTP requests/responses, the SSL pinning is correctly implemented.
+Black-box Testing can be performed by launching a MITM attack using your prefered Web Proxy to intercept [1] the traffic exchanged between client (mobile application) and the backend server. If the Proxy is unable to intercept the HTTP requests/responses, the SSL pinning is correctly implemented.
 
 #### Remediation
 
-The SSL pinning process should be implemented as described on the static analysis section.
+The SSL pinning process should be implemented as described on the static analysis section. You may find helpful in implementing certificate pinning OWASP  guide [2].
```

Comment:
Maybe write the sentence like this:
For further information please check the OWASP certificate pinning guide [2].